experimental/packages/exporter-trace-otlp-http/package.json

```diff
@@ -96,7 +96,7 @@
   "dependencies": {
     "@opentelemetry/core": "1.3.0",
     "@opentelemetry/otlp-exporter-base": "0.29.0",
-    "@opentelemetry/otlp-transformer": "0.29.0",
+    "@opentelemetry/otlp-transformer": "0.29.1",
```

Comment:
@opentelemetry/exporter-trace-otlp-http and others that depend on otlp-transformer should also be version-bumped and released, right?

Reply:
The packages' versions that are updated in this PR are overlapping with the related changed packages in https://github.com/open-telemetry/opentelemetry-js/pull/3004. Maybe we can do it in a separate PR so that we don't lose track of those versions?

Reply:
ok